src/test/java/core/basesyntax/ArrayListTest.java

```diff
@@ -1,6 +1,5 @@
 package core.basesyntax;
 
-import java.util.NoSuchElementException;
 import org.junit.Assert;
 import org.junit.Test;
```

Comment:
The 'Assert' class is used for assertions, but it is recommended to use 'assertThat' from the 'Hamcrest' library or 'assertEquals', 'assertTrue', etc., from 'JUnit' for more readability and better error messages.